### PR TITLE
removed 'check_migrations' condition in _citi_db_wrapper

### PR DESCRIPTION
**Pull request by amolgautam25**
##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

 - API

##### AWX VERSION

```

```

##### ADDITIONAL INFORMATION

```
Linked issue : https://github.com/ansible/tower/issues/5692 
After the Django upgrade these migration check are no longer needed. 
```

### CODE REVIEW

File: awx/conf/settings.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def _ctit_db_wrapper(trans_safe=False):
         yield
     except DBError as exc:
         if trans_safe:
-            if 'migrate' not in sys.argv and 'check_migrations' not in sys.argv:
+            if 'migrate' not in sys.argv:
```
Comment (jbradberry) [inline awx/conf/settings.py]: Both of these are checks concerning migrations.  The plan was to remove both of them.

**Review by jbradberry — CHANGES_REQUESTED** [review]

**Comment by shanemcd** [comment]
Why was this change made? There is no context provided and I dont see a linked issue.

**Comment by jbradberry** [comment]
> Why was this change made? There is no context provided and I dont see a linked issue.

The underlying issue is https://github.com/ansible/tower/issues/5692.  This fell out of the Django upgrade work, particularly this comment from @AlanCoding : https://github.com/ansible/awx/pull/11654#discussion_r826264003

File: awx/conf/settings.py (hunk 1 above)
Comment (amolgautam25) [reply awx/conf/settings.py]: I have removed both the conditions.

**Review by AlanCoding — APPROVED** [review]

**Review by jbradberry — APPROVED** [review]

**Pull request merged**